src/registry/registry.ts

```diff
@@ -21,7 +21,7 @@ import { inbuiltFormTransformers } from './inbuilt-components/inbuiltTransformer
  */
 export interface RegistryItem<T> {
   name: string;
-  component: T | React.ComponentType<any>;
+  component: T;
```

Comment:
@reagan-meant did you make changes to this file? Can you squash your commits to make review simpler?

Reply:
Only issue with squashing is loss of history of comments by the reviewers but sure I can do that